### ISSUE TITLE
Color mint600 is inaccessible

### ISSUE THREAD

**Issue by oddsson** [island-is/island.is#3719]
There is not enough contrast ratio between out mint600 and our white (background color), making our mint600 inaccessible. I've posted this on the design slack room, waiting for comments...

https://www.aremycolorsaccessible.com/?background=%23fff&colorCombos=%5Bobject%20Object%5D&colorCombos=%5Bobject%20Object%5D&isLight=true&textColor=%2300b39e

**Comment by oddsson** [comment]
According to the designers, the mint600 colour variant on Tags was removed from the design months ago because of this contrast issue.
